src/System.Security.Principal.Windows/ref/System.Security.Principal.Windows.cs

```diff
@@ -189,17 +187,26 @@ public enum WindowsBuiltInRole
     }
     public partial class WindowsIdentity : System.Security.Claims.ClaimsIdentity, System.IDisposable
     {
+        public new const string DefaultIssuer = "AD AUTHORITY";
```

Comment:
I would avoid exposing this const, unless we know what it would be needed for, we should probably make it private in the implementation as well if we plan not to expose it.